src/modules/users/components/GasStation/GasStationHeader/GasStationHeader.css

```diff
@@ -33,3 +35,55 @@
   composes: button from '~styles/reset.css';
   margin-left: 5px;
 }
+
+@media screen and query700 {
+  .main {
+    border-bottom-width: 1px;
+  }
+
+  .actionsContainer {
+    gap: 10px;
+  }
+
+  .closeButton {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 26px;
+    width: 26px;
+    border: 1px solid color-mod(var(--dark) alpha(0.65));
+    border-radius: 50%;
+  }
+
+  .closeButton i {
+    transform: scale(1.3);
+  }
+
+  .closeButton svg {
+    fill: color-mod(var(--dark) alpha(0.65));
+  }
+
+  /* .goToWalletIcon {
+    display: none;
+  } */
+
+  /* .walletAddress span:first-child {
+    font-size: 12px;
+    font-weight: 500;
+  } */
+
+  /* .walletAddress span:nth-child(2) {
+    display: none;
+  } */
+
+  /* .walletHeading h4 {
+    color: black;
+    letter-spacing: 0.25px;
+  } */
```

Comment:
Are those styles going to be used later? Then I would recommend leaving a note explaining that. Otherwise, you might as well delete them